### ISSUE TITLE
Wine Windows console resizing jittering

### ISSUE THREAD

**Issue by TurkeyMcMac** [wmcbrine/PDCurses#101]
I have this program:

```c
#include "curses.h"

int main(void)
{
	initscr();
	keypad(stdscr, true);
	// I'm not setting timeout(0), but this loop goes without user input
	// anyway. I guess KEY_RESIZE is being sent repeatedly.
	while (getch() != 'q') {
		if (is_termresized()) {
			resize_term(0, 0);
		}
		mvprintw(0, 0, "%dx%d\n", COLS, LINES);
		refresh();
	}
	endwin();
}
```

I compile the the wincon backend:

```
make CC=x86_64-w64-mingw32-gcc
```

I compile my program in the PDCurses root directory:

```
x86_64-w64-mingw32-gcc -Lwincon -o bug.exe bug.c -static -l:pdcurses.a
```

And I run the executable with Wine:

```
wineconsole bug.exe
```

Without me changing the console dimensions, they jitter around on their own. The dimensions tend to shrink over time, although sometimes they stabilize.

I would test this in real Windows, but I don't have access to a Windows computer.

I have tried compiling with the X11 backend and the jittering does not appear.

**Comment by Bill-Gray** [comment]
I've run into this myself for some time.  It works properly on "real" Windows and has failed with wineconsole for many versions (I'm currently using Wine 5.0 under Xubuntu 20.04).

I assume it's a Wine problem,  since it works on every "real" Windows machine I've tried (ME,  Win7,  Win10) and fails on every Wine system I've tried.  Of course,  that doesn't mean there isn't some way of tweaking the WinCon port of PDCurses to get around the issue.

It'd be nice if WinCon worked on Wine;  it would simplify some testing.  Since I can get around that by testing on a "real" Windows machine,  I haven't really dug into the problem very deeply and can't offer much more than "me too".

As you note,  the jittering does not appear on X11,  WinGUI in Wine,  SDL1/2,  or really on any combo other than WinCon and wineconsole.  I've never known it _not_ to happen with WinCon and wineconsole.
